packages/types/src/extrinsic/v4/ExtrinsicSignature.ts

```diff
@@ -13,6 +13,7 @@ import { Enum } from '../../codec/Enum';
 import { Struct } from '../../codec/Struct';
 import { EMPTY_U8A, IMMORTAL_ERA } from '../constants';
 import { GenericExtrinsicPayloadV4 } from './ExtrinsicPayload';
+import { Option } from '@polkadot/types';
```

Comment:
```suggestion
import { Option } from '../../codec/Option';
```

This is weird, yes. However the imports are direct since there are a lot of inter-deps between this stuff which creates circular deps in _some_ cases.